packages/lit-html/src/directives/render-light.ts

```diff
@@ -0,0 +1,106 @@
+import {Directive, DirectiveResult, directive, NodePart} from '../lit-html.js';
+
+export interface RenderLightHost extends HTMLElement {
+  renderLight(): unknown;
+}
+
+class RenderLight extends Directive {
```

Comment:
I added `renderLight` under `lit-html/directives`; we had this in `LitElement` in the old repo, but it's not really `LitElement`-specific _per se_, and it fits better under lit-html directives than as a one-off under `LitElement`.  TBD